strapi/src/api/pole/content-types/pole/schema.json

```
@@ -0,0 +1,57 @@
+{
+  "kind": "collectionType",
+  "collectionName": "poles",
+  "info": {
+    "singularName": "pole",
+    "pluralName": "poles",
+    "displayName": "Pole"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "pole_name": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "pole_description": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "richtext",
+      "required": true
+    },
+    "pole_logo": {
+      "allowedTypes": [
+        "images"
+      ],
+      "type": "media",
+      "multiple": false,
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "required": false
+    },
+    "members": {
+      "type": "relation",
+      "relation": "oneToMany",
```

```suggestion
      "relation": "manyToMany",
```

I think that members could be in several poles